account-kit/react/src/experimental/hooks/useSendCalls.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  clientHeaderTrack,
+  type EntryPointVersion,
+  type UserOperationRequest,
+} from "@aa-sdk/core";
+import type { GetSmartWalletClientResult } from "@account-kit/core/experimental";
+import type { SmartWalletClient } from "@account-kit/wallet-client";
+import {
+  useMutation,
+  type UseMutateAsyncFunction,
+  type UseMutateFunction,
+} from "@tanstack/react-query";
+import { sendTransaction as wagmi_sendTransaction } from "@wagmi/core";
+import { fromHex, type Address, type Hex, type JsonRpcAccount } from "viem";
+import { useAccount as wagmi_useAccount } from "wagmi";
+import {
+  ClientUndefinedHookError,
+  UnsupportedEOAActionError,
+} from "../../errors.js";
+import { useAlchemyAccountContext } from "../../hooks/useAlchemyAccountContext.js";
+import { ReactLogger } from "../../metrics.js";
+
+export type UseSendCallsParams = {
+  client?: GetSmartWalletClientResult<Address>;
+};
+
+type MutationResult<
+  TEntryPointVersion extends EntryPointVersion = EntryPointVersion,
+> = {
+  id: Hex;
+  // TODO: deprecate this
```

Comment:
actually... not sure if we should deprecate this. we will need it for drop and replace unless we come up with a better API for drop and replace